ccmlib/scylla_cluster.py

```diff
@@ -475,6 +475,8 @@ def agent_download_files(self, node, location_list, snapshot_tag, keyspace_filte
                 "--mode", "upload", "-T", snapshot_tag, "-d", os.path.join(node.get_path(), "data")]
         if keyspace_filter_list:
             args.extend(["-K", ",".join(keyspace_filter_list)])
+        if dry_run:
+            args.append("--dry-run")
```

Comment:
could you please add either a log print or pydoc explaining what will happen when running this dry-run?

Reply:
Done